test/e2e/instrumentation/core_events.go

```diff
@@ -44,95 +44,17 @@ var _ = common.SIGDescribe("Events", func() {
 	f.NamespacePodSecurityEnforceLevel = admissionapi.LevelPrivileged
 
 	/*
-			   Release: v1.20
```

Comment:
are we removing something? by design?

Reply:
Can we put that it in a comment in the code?  So if by some horrific accident we forget, someone can fix it later?

Reply:
+1 @smarterclayton